extensions/admin-tool-ext-win/yarn.lock

```diff
@@ -1213,10 +1213,10 @@ md5@^2.1.0:
     crypt "~0.0.1"
     is-buffer "~1.1.1"
 
-minimatch@3.0.4, minimatch@^3.0.4:
```

Comment:
It looks like the extensions also have the same pinned version problem. In this case it's from mocha : 

```
mocha@^7.1.1:
  version "7.2.0"
  resolved "https://registry.yarnpkg.com/mocha/-/mocha-7.2.0.tgz#01cc227b00d875ab1eed03a75106689cfed5a604"
  integrity sha512-O9CIypScywTVpNaRrCAgoUnJgozpIofjKUYmJhiCIJMiuYnLI6otcb1/kpW9/n/tJODHGZ7i8aLQoDVsMtOKQQ==
  dependencies:
...
    minimatch "3.0.4"
```

so at this point I'd say that it'd be better to just go through and see if the vulnerable version of minimatch is actually used in any of our runtime dependencies. If it is then we can selectively fix those - but all these other ones we can just dismiss.